pkg/server/event_handler_test.go

```diff
@@ -176,18 +211,20 @@ func TestEventHandler_Handle(t *testing.T) {
 	t.Parallel()
 
 	cases := []struct {
-		name             string
-		notification     pubsub.Message
-		gcsObjectBytes   []byte
-		processors       []Processor[*structpb.Struct]
-		successMessenger Messenger
-		failureMessenger Messenger
-		wantErrSubstr    string
+		name              string
+		notification      pubsub.Message
+		gcsObjectBytes    []byte
+		githubSourceBytes []byte
```

Comment:
Not sure why it's `[]byte` because we are not validating it anywhere

Reply:
This is removed now, as we no longer needs this.